starklings.py

```diff
@@ -48,13 +47,19 @@ def is_valid_file(parser, arg):
 )
 
 root_parser.add_argument(
-    "-s",
     "--solution",
+    "-s",
     metavar="relative_path_to_exercise",
     help="Provide a solution for an exercise",
     type=lambda x: is_valid_file(root_parser, x),
 )
 
+root_parser.add_argument(
+    "--display-course",
```

Comment:
Minor: add a `-d` for consistency and speed aficionados

Reply:
done